admin/src/test/java/com/exadel/frs/AppServiceTest.java

```diff
@@ -576,6 +573,98 @@ void successPassAllOwnedAppsToNewOwnerAndLeave() {
+    void getAppRolesToAssignReturnsToOwner() {
+        val user = user(USER_ID);
+        val organization = organization();
+
+        val app = App.builder()
+                .id(APPLICATION_ID)
+                .guid(APPLICATION_GUID)
+                .organization(organization)
+                .build();
+        app.addUserAppRole(user, OWNER);
+
+        when(appRepositoryMock.findByGuid(APPLICATION_GUID)).thenReturn(Optional.of(app));
+
+        val result = appService.getAppRolesToAssign(ORGANISATION_GUID, APPLICATION_GUID, USER_ID);
+
+        assertThat(result.length).isNotZero();
+
+        verify(authManagerMock).verifyReadPrivilegesToApp(USER_ID, app);
+        verify(authManagerMock).verifyOrganizationHasTheApp(ORGANISATION_GUID, app);
+        verifyNoMoreInteractions(authManagerMock);
+    }
+
+    @Test
+    void getAppRolesToAssignReturnsToUser() {
+        val user = user(USER_ID);
+        val organization = organization();
+        organization.addUserOrganizationRole(user, USER);
+
+        val app = App.builder()
+                .id(APPLICATION_ID)
+                .guid(APPLICATION_GUID)
+                .organization(organization)
+                .build();
+
+        when(appRepositoryMock.findByGuid(APPLICATION_GUID)).thenReturn(Optional.of(app));
+
+        val result = appService.getAppRolesToAssign(ORGANISATION_GUID, APPLICATION_GUID, USER_ID);
+
+        assertThat(result.length).isZero();
+
+        verify(authManagerMock).verifyReadPrivilegesToApp(USER_ID, app);
+        verify(authManagerMock).verifyOrganizationHasTheApp(ORGANISATION_GUID, app);
+        verifyNoMoreInteractions(authManagerMock);
+    }
+
+    @Test
+    void getAppRolesToAssignReturnsDefault() {
+        val user = user(USER_ID);
+        val organization = organization();
+        organization.addUserOrganizationRole(user, OrganizationRole.OWNER);
+
+        val app = App.builder()
+                .id(APPLICATION_ID)
+                .guid(APPLICATION_GUID)
+                .organization(organization)
+                .build();
+
+        when(appRepositoryMock.findByGuid(APPLICATION_GUID)).thenReturn(Optional.of(app));
+
+        val result = appService.getAppRolesToAssign(ORGANISATION_GUID, APPLICATION_GUID, USER_ID);
+
+        assertThat(result.length).isNotZero();
+
+        verify(authManagerMock).verifyReadPrivilegesToApp(USER_ID, app);
+        verify(authManagerMock).verifyOrganizationHasTheApp(ORGANISATION_GUID, app);
+        verifyNoMoreInteractions(authManagerMock);
+    }
+
+    @Test
+    void getAppRolesToAssignThrowsExceptionIfNoOrgUser() {
+        val user = user(USER_ID);
+        val organization = organization();
+
+        val app = App.builder()
+                .id(APPLICATION_ID)
+                .guid(APPLICATION_GUID)
+                .organization(organization)
+                .build();
+
+        when(appRepositoryMock.findByGuid(APPLICATION_GUID)).thenReturn(Optional.of(app));
+
+        assertThrows(
```

Comment:
replace all from assertThrows to org.assertj.core.api.Assertions.assertThatThrownBy method call please